aioxmpp/e2etest/provision.py

```diff
@@ -287,6 +290,12 @@ def get_connected_client(self, presence=aioxmpp.PresenceState(True),
         Clients obtained from this function are cleaned up automatically on
         tear down of the test. The clients are stopped and the accounts
         deleted or cleared, so that each test starts with a fully fresh state.
+
+        A coroutine may be passed as `prepare` argument. It is called
```

Comment:
Technically, it is a coroutinefunction. I always mix this up, too, and we need to go through the docs and clean this up at some point.